src/pubsub/message/index.js

```diff
@@ -1,17 +1,16 @@
 'use strict'
 
+// @ts-ignore protons not typed
 const protons = require('protons')
 
 const rpcProto = protons(require('./rpc.proto.js'))
 const RPC = rpcProto.RPC
 const topicDescriptorProto = protons(require('./topic-descriptor.proto.js'))
 
-/**
- * @module pubsub/message/index
- */
-exports = module.exports
-exports.rpc = rpcProto
-exports.td = topicDescriptorProto
-exports.RPC = RPC
-exports.Message = RPC.Message
-exports.SubOpts = RPC.SubOpts
+module.exports = {
```

Comment:
out of interest: was this change forced by the type checking?

Reply:
By the linter, but I think this can cause problems with inferred types and some bundlers. @hugomrdias has the bigger picture here on why we should avoid this